rpcs3/Emu/Cell/SPURecompiler.cpp

```diff
@@ -34,6 +34,7 @@ extern const spu_decoder<spu_interpreter_fast> g_spu_interpreter_fast;
 
 extern u64 get_timebased_time();
 
+
```

Comment:
inadvertent whitespace. You can just delete it with github UI and we can squash when merging